src/geocoding/slf/tests/test_locparser.py

```diff
@@ -0,0 +1,222 @@
+import pytest
+from geocoding.slf.locparser import *
+
+
+@pytest.mark.parametrize('ngsi_geom_type', [
+    None, '', 'geo:???', 'geo: ', 'geo: line'  # note the space!
+])
+def test_lookup_parser_return_fail_parser_on_unknown_geom(ngsi_geom_type):
+    parser = lookup_parser(ngsi_geom_type)
+    assert parser is not None
+
+    entity_with_valid_location = {
+        'location': {
+            'value': '41.3763726, 2.186447514',
+            'type': 'geo:point'
+        }
+    }
+    parsed = parser(entity_with_valid_location)
+    assert parsed is None
```

Comment:
if the parser simply returns `None`, who's the one telling the user "wrong param dude!" again?

Reply:
like we discussed in the call, we'll do that as part of error handling in the http response.